README.md

```diff
@@ -0,0 +1,68 @@
+# 3D Pinball for Windows – Space Cadet
```

Comment:
In general, I'm against of READMEs like this as they just duplicate existing documentation.

Reply:
Removed the description file, only kept the flatpak part

Reply:
It's kinda like this: https://github.com/flathub/com.jetbrains.PyCharm-Community
I have a Makefile for easier flatpak build and check (but it's optional)